Src/WitsmlExplorer.Frontend/services/credentialsService.ts

```diff
@@ -20,12 +23,11 @@ export interface AuthorizationState {
   status: AuthorizationStatus;
 }
 
-class CredentialsService {
-  private static _instance: CredentialsService;
+class AuthorizationService {
```

Comment:
I forgor to change the file name to authorizationService. 💀
This is fixed in #1726.